SqlInstall.ps1

```diff
@@ -0,0 +1,433 @@
+            If ([string]::IsNullOrEmpty($NewBackupVolume)) {
+                Write-Message -Level Verbose -Message "BackupVolume remains on " $BackupVolume
+            }
+            else {
+                $NewBackupVolume = $NewBackupVolume -replace "\\$"
+                $NewBackupVolume = $NewBackupVolume -replace ":$"
+                $BackupVolume = $NewBackupVolume
+                Write-Message -Level Verbose -Message "BackupVolume moved to " $BackupVolume
+            }
+        }
+        Default {Write-Message -Level Verbose -Message "Drives agreed, continuing"; }
+    }
+
+
+    # Out-File -FilePath C:\Temp\ConfigurationFile2.ini -InputObject $startScript
+
+    (Get-Content -Path $configini).Replace('SQLBACKUPDIR="E:\Program Files\Microsoft SQL Server\MSSQL12.AXIANSDB01\MSSQL\Backup"', 'SQLBACKUPDIR="' + $BackupVolume + ':\Program Files\Microsoft SQL Server\MSSQL12.AXIANSDB01\MSSQL\Backup"') | Out-File $configini
+
+    (Get-Content -Path $configini).Replace('SQLUSERDBDIR="E:\Program Files\Microsoft SQL Server\MSSQL12.AXIANSDB01\MSSQL\Data"', 'SQLUSERDBDIR="' + $DataVolume + ':\Program Files\Microsoft SQL Server\MSSQL12.AXIANSDB01\MSSQL\Data"') | Out-File $configini
+
+    (Get-Content -Path$configini).Replace('SQLTEMPDBDIR="E:\Program Files\Microsoft SQL Server\MSSQL12.AXIANSDB01\MSSQL\Data"', 'SQLTEMPDBDIR="' + $TempVolume + ':\Program Files\Microsoft SQL Server\MSSQL12.AXIANSDB01\MSSQL\Data"') | Out-File $configini
+
+    (Get-Content -Path $configini).Replace('SQLUSERDBLOGDIR="E:\Program Files\Microsoft SQL Server\MSSQL12.AXIANSDB01\MSSQL\Log"', 'SQLUSERDBLOGDIR="' + $LogVolume + ':\Program Files\Microsoft SQL Server\MSSQL12.AXIANSDB01\MSSQL\Log"') | Out-File $configini
+
+    (Get-Content -Path $configini).Replace('SQLSYSADMINACCOUNTS="WIN-NAJQHOBU8QD\Administrator"', 'SQLSYSADMINACCOUNTS="' + $SqlServerAccount)| Out-File $configini
+
+    If ($SaveFile -eq "Yes") {
+        $SaveFileLocation = Read-Host "Please enter your preferred directory for saving a copy of the configuration file: "
+        Copy-Item $configini -Destination $SaveFileLocation
+    }
+
+    IF ($Authentication -eq "Mixed Mode") {
+        $SAPassW = [PsCredential](Get-Credential -UserName "SA"  -Message "Please Enter the SA Password.")
+
+        & $SetupFile /ConfigurationFile=$configini /Q /IACCEPTSQLSERVERLICENSETERMS /SAPWD= $SAPassW.GetNetworkCredential().Password
+
+    }
+    else {
+        & $SetupFile /ConfigurationFile=$configini /Q /IACCEPTSQLSERVERLICENSETERMS
+    }
+
+    
+
+    # Grant service account the right to perform volume maintenance
+    # code found at https://social.technet.microsoft.com/Forums/windows/en-US/5f293595-772e-4d0c-88af-f54e55814223/adding-domain-account-to-the-local-policy-user-rights-assignment-perform-volume-maintenance?forum=winserverpowershell
+
+    if ($PerformVolumeMaintenance) {
+        ## <--- Configure here
+        $accountToAdd = 'NT Service\MSSQL$AXIANSDB01'
```

Comment:
This would be `"NT Service\MSSQL`$$env:ComputerName"` (or whatever syntax works ;))